samples/mpu/mpu_stack_guard_test/sample.yaml

```diff
@@ -5,7 +5,7 @@ tests:
     tags: apps
     harness: console
     # We do not handle segfaults here
-    arch_exclude: posix riscv32 nios2
+    arch_exclude: posix riscv32 nios2 xtensa
```

Comment:
This change is fine. I need to fix this filter so that it instead filters on whether the CPU has stack protection features.